mdit_py_plugins/attrs/index.py

```diff
@@ -58,6 +61,11 @@ def _attr_inline_rule(state: StateInline, silent: bool) -> bool:
             return False
         try:
             new_pos, attrs = parse(state.src[state.pos :])
+            if allowed_attributes is not None:
+                attrs = {k: v for k, v in attrs.items() if k in allowed_attributes}
+                token.meta["insecure_attrs"] = {
+                    k: v for k, v in attrs.items() if k not in allowed_attributes
+                }
```

Comment:
```suggestion
            if allowed_attributes is not None and (disallowed := {
                    k: v for k, v in attrs.items() if k not in allowed_attributes
                }):
                token.meta["insecure_attrs"] = dissalowed
                attrs = {k: v for k, v in attrs.items() if k in allowed_attributes}
```

you were tryting to take the insecure after you'd already removed them 😅